packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/ListingType/ListingType.tsx

```diff
@@ -28,7 +26,6 @@ export const ListingType: FC<ListingTypeProps> = ({ selectedType, onSelectType }
         <OptionCardGroupRadio
           selectedValue={selectedType}
           onChange={(value) => onSelectType(value as Listing)}
-          direction={mdMatch ? 'horizontal' : 'vertical'}
```

Comment:
This didn't change anything, I guess `horizontal` is the default value?
![Screenshot 2022-06-24 at 08 53 43](https://user-images.githubusercontent.com/12646744/175479780-4faa7d6c-c88e-4bce-ac79-7f91f4332f2f.jpg)